version/version.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // The main version number that is being run at the moment.
-var Version = "1.4.3"
+var Version = "1.4.4"
```

Comment:
I might be misreading but I thought we should update _only_ version/VERSION and leave this for the automation to increment.

Reply:
yep, reverted!

Reply:
No automation updated the versions in version.go, before the Git tag was made... was it actually correct to not update this?

Reply:
Yes we should have updated this. I've updated our internal release documentation to make this clear.